<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditor</class>
 <widget class="QWidget" name="FormEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>230</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Новая клавиша</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Клавиша</string>
   </property>
   <widget class="QLineEditExtForKey" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите...</string>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Контроллер</string>
   </property>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Master</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Extension 8</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Вращение</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;--</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>--&gt;</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Мотор</string>
   </property>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>M1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>M4</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>211</width>
     <height>54</height>
    </rect>
   </property>
   <property name="title">
    <string>Скорость</string>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>16</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>32</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>512</number>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>171</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>171</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLineEditExtForKey</class>
   <extends>QLineEdit</extends>
   <header location="global">qlineeditextforkey.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <signal>closed()</signal>
 </slots>
</ui>
